main.go

```diff
@@ -51,7 +51,7 @@ func main() {
 		os.Exit(1)
 	}
 
-	load := []string{filepath.Join(path, "picotool"), "load", *binary + ".uf2"}
+	load := []string{filepath.Join(path, "picotool"), "load -v", *binary + ".uf2"}
```

Comment:
```suggestion
	load := []string{filepath.Join(path, "picotool"), "load", "-v", *binary + ".uf2"}
```

The previous code produces an invalid command (i.e., `picotool "load -v"`):

```text
Unknown command: load -v
```